packages/react/src/Breakout/BreakoutCell.tsx

```diff
@@ -21,9 +26,9 @@ type BreakoutCellSpanAllProp = {
 
 type BreakoutCellSpanAndStartProps = {
   /** The amount of grid columns the cell spans. */
-  colSpan?: 'all' | GridColumnNumber | GridColumnNumbers
+  colSpan?: 'all' | GridNarrowColumnNumber | GridMediumColumnNumber | GridWideColumnNumber | GridColumnNumbers
```

Comment:
This has overlap: 

We discussed `Pick<>`, but that’s for object types, not literals.

This may be an option:

```
type Subset<T, U> = T extends U ? T : never

export type GridColumnNumber = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12

export type GridColumnNumbers = {
  narrow: Subset<GridColumnNumber, 1 | 2 | 3 | 4>
  medium: Subset<GridColumnNumber, 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8>
  wide: GridColumnNumber
}
```

Types that accept all three variations can keep using `GridColumnNumber`.

Reply:
This is an alternative, but seems overkill:

```ts
type Range<N extends number, Result extends Array<unknown> = []> = Result['length'] extends N
  ? Result[number]
  : Range<N, [...Result, Result['length']]>

export type GridColumnNumber = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12

export type GridColumnNumbers = {
  narrow: Range<4>
  medium: Range<8>
  wide: GridColumnNumber
}
```

Reply:
Subset seems best?

Reply:
Well, I like how simple it is to use `Range` even if the definition is more complex.

I did something similar in the experimental branch: https://github.com/Amsterdam/design-system/blob/9cad4c476dba00f5ec985a4b6fb538758ca0304f/packages/react/src/Grid/Grid.tsx#L12

Reply:
In the end chose the KISS solution. :-)